packages/patterns/list-operations.tsx

```diff
@@ -0,0 +1,157 @@
+/// <cts-enable />
+import {
+  Cell,
+  Default,
+  h,
+  handler,
+  NAME,
+  derive,
+  str,
+  Opaque,
+  recipe,
+  lift,
+  toSchema,
+  ID,
+  UI,
+} from "commontools";
+
+interface Item {
+  [ID]: number;
+  title: string;
+}
+
+interface ListInput {
+  items: Default<Item[], []>;
+}
+
+interface ListOutput extends ListInput {}
+
+const typeTest = handler((_, state: { a: Cell<Item[]>, b: readonly Item[], c: readonly Cell<Item>[], d: Cell<Cell<Item>[]> }) => {
+  const { a, b, c, d } = state;
+  console.log({ a, b, c, d })
+});
+
+const resetList = handler((_, state: { items: Cell<Item[]> }) => {
+  state.items.set([{ [ID]: Math.random(), title: "A" }, { [ID]: Math.random(), title: "B" }, { [ID]: Math.random(), title: "C" }, { [ID]: Math.random(), title: "D" }]);
```

Comment:
You don't have the use Math.random here, the event context is already unique and ID have to be unique only per context

In a lift the context is stable across calls, but to make calls idempotent you want predictable ID is well.

So I'd rather use some predictable ID as example here as well.

(And FWIW, if you .push those, then no ID is necessary. I want to make that true for .set as well)